src/components/ColorPicker/ColorPicker.st.css

```diff
@@ -16,6 +16,7 @@
     border: 0;
     padding: 0.01em 0 0 0;
     min-width: 0;
+    user-select: none;
```

Comment:
Won't this disable the onClick?

Reply:
onClick and onChange are still working.
`user-select: none` is needed in order to fix a bug on iOS when doing long press. We want to show a tooltip but system text selection menu is shown instead.